sys-libs/libxcrypt/libxcrypt-4.4.28.ebuild

```diff
@@ -177,6 +216,12 @@ src_install() {
 	# Remove useless stuff from installation
 	find "${ED}"/usr/share/doc/${PF} -type l -delete || die
 	find "${ED}" -name '*.la' -delete || die
+
+	# workaround broken upstream cross-* --docdir by installing files in proper locations
+	if is_cross; then
+		install -d "${ED}$(get_xcprefix)"/usr/share
+		mv "${ED}"/usr/share/doc "${ED}$(get_xcprefix)"/usr/share
```

Comment:
This won't matter I _think_ once using `insinto`/`doins`, but _second_ argument should have `${D}` not `${ED}` for earlier mentioned reasons.

Also, use `|| die` on external commands.